### PR TITLE
Changes docs to deprecate the K8s dashboard data ingest method

### LINKED ISSUE #3711: Update documentation for getting data into the Kubernetes Dashboard
## Description 

We are going to deprecate the currently documented way of setting up the Kubernetes dashboard, in favor of a method that uses the D4C integration. This requires changes to the D4C docs and the Dashboards docs.

## Acceptance Test Criteria

Approved by Nick and Security Docs

### PR DESCRIPTION
**Pull request by benironside**
Fixes #3711 by changing the method described in docs for getting data into the Kubernetes dashboard. The new method is to use the D4C integration. 

Previews: 
- [Get started with D4C ](https://security-docs_3775.docs-preview.app.elstc.co/guide/en/security/master/d4c-get-started.html)
- [Kubernetes dashboard](https://security-docs_3775.docs-preview.app.elstc.co/guide/en/security/master/kubernetes-dashboard.html)

@nick-alayil , please especially review the requirements section on the get started with container workload protection page. It's currently a direct clone from the Kubernetes dashboard page, and I'm sure it will need to be updated.

### CODE REVIEW

File: docs/cloud-native-security/d4c-get-started.asciidoc [hunk 1]
```diff
@@ -6,7 +6,14 @@
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [get-started]
 
-This page describes how to set up Container Workload Protection (CWP) for various use cases.
+This page describes how to set up Container Workload Protection (CWP).
```
Comment (nick-alayil) [inline docs/cloud-native-security/d4c-get-started.asciidoc:9]: >Container Workload Protection (CWP).

Cloud Workload Protection for Kubernetes.

File: docs/dashboards/kubernetes-dashboard.asciidoc [hunk 2]
```diff
@@ -34,74 +34,29 @@ The *Metadata* tab is organized into these expandable sections:
 - *Container:* `id`, `name`, `image.name`, `image.tag`, `image.hash.all`
 - *Orchestrator:* `resource.ip`, `resource.name`, `resource.type`, `namespace`, `cluster.id`, `cluster.name`, `parent.type`
 
-
 [discrete]
-[[k8s-dash-setup]]
 == Setup
-To collect session data for the dashboard, you'll deploy a Kubernetes DaemonSet to your clusters that implements the {elastic-defend} integration.
+To get data for this dashboard, set up <<d4c-get-started, Container Workload Protection>> for the clusters you want to appear on the dashboard.
```
Comment (nick-alayil) [inline docs/dashboards/kubernetes-dashboard.asciidoc:39]: > Container Workload Protection

Cloud Workload Protection for Kubernetes

**Review by nick-alayil — APPROVED** [review]
LGTM. Just some minor correction on changing Container Workload Protection to Cloud Workload Protection for Kubernetes. Thanks Ben.

File: docs/cloud-native-security/cloud-nat-sec-kubernetes-dashboard.asciidoc [hunk 3]
```diff
@@ -33,74 +33,29 @@ The *Metadata* tab is organized into these expandable sections:
 - *Container:* `id`, `name`, `image.name`, `image.tag`, `image.hash.all`
 - *Orchestrator:* `resource.ip`, `resource.name`, `resource.type`, `namespace`, `cluster.id`, `cluster.name`, `parent.type`
 
-
 [discrete]
-[[cloud-nat-sec-k8s-dash-setup]]
 == Setup
-To collect session data for the dashboard, you'll deploy a Kubernetes DaemonSet to your clusters that implements the {elastic-defend} integration.
+To get data for this dashboard, set up <<d4c-get-started, Cloud Workload Protection for Kubernetes>> for the clusters you want to appear on the dashboard.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/cloud-nat-sec-kubernetes-dashboard.asciidoc:38]:
```suggestion
To get data for this dashboard, set up <<d4c-get-started, Cloud Workload Protection for Kubernetes>> for the clusters you want to display on the dashboard.
```

File: docs/cloud-native-security/d4c-overview.asciidoc [hunk 4]
```diff
@@ -28,15 +28,15 @@ CWP uses a powerful policy language to restrict container workloads to a set of
...
 | Network blocking | ✗ | ✗
 | Drift prevention | ✓ | ✓
 | Mount point awareness | ✓ | ✓
 |===
 
 [discrete]
-== How CWP works
-CWP uses a lightweight integration, Defend for Containers (D4C). When you set up the D4C integration, it gets deployed by {agent}. Specifically, the {agent} gets installed as a DaemonSet on your Kubernetes clusters, where it enables D4C to use eBPF Linux Security Modules https://docs.kernel.org/bpf/prog_lsm.html[LSM] and tracepoint probes to record system events. Events are evaluated against LSM hook points, enabling {agent} to evaluate system activity against your policy before allowing it to proceed.
+== How CWP for Kubernetes works
+CWP for Kubernetes uses a lightweight integration, Defend for Containers (D4C). When you set up the D4C integration, it gets deployed by {agent}. Specifically, the {agent} gets installed as a DaemonSet on your Kubernetes clusters, where it enables D4C to use eBPF Linux Security Modules https://docs.kernel.org/bpf/prog_lsm.html[LSM] and tracepoint probes to record system events. Events are evaluated against LSM hook points, enabling {agent} to evaluate system activity against your policy before allowing it to proceed.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/d4c-overview.asciidoc:39]: Should the LSM acronym be in parentheses? 

```suggestion
CWP for Kubernetes uses a lightweight integration, Defend for Containers (D4C). When you set up the D4C integration, it gets deployed by {agent}. Specifically, the {agent} is installed as a DaemonSet on your Kubernetes clusters, where it enables D4C to use eBPF Linux Security Modules https://docs.kernel.org/bpf/prog_lsm.html[LSM] and tracepoint probes to record system events. Events are evaluated against LSM hook points, enabling {agent} to evaluate system activity against your policy before allowing it to proceed.
```

File: docs/dashboards/kubernetes-dashboard.asciidoc [hunk 5]
```diff
@@ -34,74 +34,29 @@ The *Metadata* tab is organized into these expandable sections:
 - *Container:* `id`, `name`, `image.name`, `image.tag`, `image.hash.all`
 - *Orchestrator:* `resource.ip`, `resource.name`, `resource.type`, `namespace`, `cluster.id`, `cluster.name`, `parent.type`
 
-
 [discrete]
-[[k8s-dash-setup]]
 == Setup
-To collect session data for the dashboard, you'll deploy a Kubernetes DaemonSet to your clusters that implements the {elastic-defend} integration.
+To get data for this dashboard, set up <<d4c-get-started, Cloud Workload Protection for Kubernetes>> for the clusters you want to appear on the dashboard.
```
Comment (nastasha-solomon) [inline docs/dashboards/kubernetes-dashboard.asciidoc:39]:
```suggestion
To get data for this dashboard, set up <<d4c-get-started, Cloud Workload Protection for Kubernetes>> for the clusters you want to display on the dashboard.
```

**Review by nastasha-solomon — APPROVED** [review]
Couple small nits. Everything else looks good!

File: docs/cloud-native-security/cloud-nat-sec-kubernetes-dashboard.asciidoc [hunk 6]
```diff
@@ -33,74 +33,29 @@ The *Metadata* tab is organized into these expandable sections:
...
 .Requirements
 [sidebar]
 --
-- Session data capture requires a Linux kernel version of 5.10.16 or higher. Session View does not support older kernels.
-- This feature requires Elastic Stack version 8.4 or newer.
-- You need an active {fleet-guide}/fleet-overview.html[{fleet} Server].
-- Your Elastic deployment must have the {elastic-defend} integration <<install-endpoint,enabled>>.
-- The {elastic-defend} integration policy must have **Include session data** set to `true`. To modify this setting, go to **Manage -> Policies**, select your policy, and find `Include session data` near the bottom of the `Policy settings` tab.
+- Kubernetes node operating systems must have Linux kernels 5.10.16 or higher.
+- Elastic Stack version 8.8 or higher.
```
Comment (joepeeples) [inline docs/cloud-native-security/cloud-nat-sec-kubernetes-dashboard.asciidoc:44]:
```suggestion
- {stack} version 8.8 or higher.
```

File: docs/cloud-native-security/d4c-get-started.asciidoc [hunk 7]
```diff
@@ -1,12 +1,19 @@
...
 :frontmatter-tags-user-goals: [get-started]
 
-This page describes how to set up Container Workload Protection (CWP) for various use cases.
+This page describes how to set up Cloud Workload Protection (CWP) for Kubernetes.
+
+.Requirements
+[sidebar]
+--
+- Kubernetes node operating systems must have Linux kernels 5.10.16 or higher.
+- Elastic Stack version 8.8 or higher.
```
Comment (joepeeples) [inline docs/cloud-native-security/d4c-get-started.asciidoc:15]:
```suggestion
- {stack} version 8.8 or higher.
```

File: docs/dashboards/kubernetes-dashboard.asciidoc [hunk 8]
```diff
@@ -34,74 +34,29 @@ The *Metadata* tab is organized into these expandable sections:
...
 .Requirements
 [sidebar]
 --
-- Session data capture requires a Linux kernel version of 5.10.16 or higher. Session View does not support older kernels.
-- This feature requires Elastic Stack version 8.4 or newer.
-- You need an active {fleet-guide}/fleet-overview.html[{fleet} Server].
-- Your Elastic deployment must have the {elastic-defend} integration <<install-endpoint,enabled>>.
-- The {elastic-defend} integration policy must have **Include session data** set to `true`. To modify this setting, go to **Manage -> Policies**, select your policy, and find `Include session data` near the bottom of the `Policy settings` tab.
+- Kubernetes node operating systems must have Linux kernels 5.10.16 or higher.
+- Elastic Stack version 8.8 or higher.
```
Comment (joepeeples) [inline docs/dashboards/kubernetes-dashboard.asciidoc:45]:
```suggestion
- {stack} version 8.8 or higher.
```

**Review by joepeeples — COMMENTED** [review]
One edit to use the `{stack}` attribute in a few spots, otherwise LGTM!

**Pull request merged**